src/padplus-manager/padplus-manager.ts

```diff
@@ -477,6 +477,11 @@ export class PadplusManager extends EventEmitter {
                 log.verbose(PRE, `init cache manager`)
                 await CacheManager.init(wechatUser.userName)
                 this.cacheManager = CacheManager.Instance
+                if (this.padplusUser) {
+                  await this.padplusUser.reconnect()
+                } else {
+                  throw new Error(`no padplus user.`)
```

Comment:
Let's not throw error here if we failed to send the reconnect request. Put a log here like below:
```ts
log.info(PRE, `Failed to force sync the account with server, might cause server stuck`)
```

Reply:
This error is not related to Reconnect API, it will catch error that happened in `start` method for `padplusUser` init. I think it is a bug that need to catch.

When we use class like `padplusContact`, `padplusUser` and so on, we should check it existed or not first.